### PR TITLE
fix: openid4vci holder jwt_vc_json-ld missing context

### PR DESCRIPTION
**Pull request by Sshovon**
This PR fixes openid4vci holder module jwt_vc_json-ld and ldp_vc missing context issue during credential build. I have described the whole issue  [here](https://github.com/openwallet-foundation/credo-ts/issues/2044).

### CODE REVIEW

File: packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts
Comment (TimoGlastra) [inline packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts:494]: I don't think the cast is needed

```suggestion
      // retrieve the @context from the credential configuration
      const context = offeredCredentialConfiguration.format === OpenId4VciCredentialFormatProfile.JwtVcJsonLd || offeredCredentialConfiguration.format === OpenId4VciCredentialFormatProfile.LdpVc ? offeredCredentialConfiguration.credential_definition['@context'] : undefined
```

File: packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts
Comment (TimoGlastra) [inline packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts:26]:
```suggestion
```

**Review by TimoGlastra — COMMENTED** [review]
Thanks for the PR 🙌

**Review by TimoGlastra — APPROVED** [review]

**Comment by TimoGlastra** [comment]
Can you sign-off your commit? I can't merge it otherwise

https://github.com/openwallet-foundation/credo-ts/pull/2047/checks?check_run_id=31280956610

**Comment by Sshovon** [comment]
Hey @TimoGlastra, I have added the sign-off. Thanks.

**Comment by TimoGlastra** [comment]
Can you address the ci issues (linting)? Otherwise this is good to go!

**Comment by Sshovon** [comment]
Hey @TimoGlastra , I have added the linting fix.

**Comment by TimoGlastra** [comment]
Thanks a lot!!

**Comment by TimoGlastra** [comment]
Thanks a lot!!

**Pull request merged**
